kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,16 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must complete the Operator installation and then create the required custom resources (CRs).
+
+NOTE: {KubeFedProductName} can be deployed as either a namespace-scoped or a cluster-scoped deployment, but not both, as this will cause conflicts when both deployments attempt to manage the same set of resources.
+
+== Installing the {KubeFedProductName} Operator
+You can install the {KubeFedProductName} Operator in the host cluster by following the {product-title} instructions on installing an Operator.
```

Comment:
Seems we do not have a doc to explain what is host cluster, suggest we can add some terms about kubefed.

Reply:
This would be added to an architecture module - I will let you know when there are updates on this.